src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/TransitivelyReferencedPackageModel.cs

```diff
@@ -5,16 +5,23 @@
 
 using System;
 using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
 using NuGet.Packaging;
 using NuGet.Packaging.Core;
+using NuGet.Protocol;
+using NuGet.Protocol.Model;
+using NuGet.VisualStudio.Internal.Contracts;
 
 namespace NuGet.PackageManagement.UI.Models
 {
-    internal class TransitivelyReferencedPackageModel : ReferencedPackageModel
+    internal class TransitivelyReferencedPackageModel : PackageModel, IDeprecationCapable, IVulnerableCapable
```

Comment:
If we're no longer extending ReferencedPackageModel, we don't have a reason to make Transitive Packages have a Deprecation Capability. Transitive Packages are not deprecation capable. This would also allow us to delete the NoDeprecationCapability and its tests.